packages/core/src/index.ts

```diff
@@ -4,7 +4,7 @@ export {
   default as reducer,
   initialState,
 } from '@rest-hooks/core/state/reducer';
-export { useDenormalized } from '@rest-hooks/core/state/selectors';
+export { useDenormalized } from '@rest-hooks/core/state/selectors/index';
```

Comment:
for this to work with or without the index, you’d need to add this exact specifier to the core package’s exports field

Reply:
The tooling remaps this to a relative import (as it has before). This is how we make typescript happy and allow babel to build various targets

Reply:
this turns into `from './state/selectors/index.js'`

before it was `from './state/selectors.js'` - but selectors is a directory so it didn't work

Reply:
ahhhh ok, i misunderstood. when using the package name, in node itself, it goes through "exports" (it's https://www.npmjs.com/package/has-package-self-reference) so this pattern looks like it's supposed to be a specifier in "exports".